definitions/npm/@ginkgo-bioworks/react-json-schema-form-builder_v1.x.x/flow_v0.30.x-v0.103.x/react-json-schema-form-builder_v1.x.x.js

```diff
@@ -0,0 +1,120 @@
+declare module '@ginkgo-bioworks/react-json-schema-form-builder' {
+  declare type CardProps = {
+    name: string,
+    required: boolean,
+    dataOptions: { [string]: any },
+    uiOptions: { [string]: any },
+    $ref?: string,
+    dependents?: Array<{
+      children: Array<string>,
+      value?: any,
+    }>,
+    dependent?: boolean,
+    parent?: string,
+    propType: string,
+    neighborNames: Array<string>,
+  };
+
+  declare type SectionProps = {
+    name: string,
+    required: boolean,
+    schema: { [string]: any },
+    uischema: { [string]: any },
+    $ref?: string,
+    dependents?: Array<{
+      children: Array<string>,
+      value?: any,
+    }>,
+    dependent?: boolean,
+    propType: string,
+    neighborNames: Array<string>,
+  };
+
+  declare type ElementProps = CardProps & SectionProps;
+
+  declare type Parameters = {
+    [string]: string | number | boolean | Array<string | number>,
+    name: string,
+    path: string,
+    definitionData: { [string]: any },
+    definitionUi: { [string]: any },
+    category: string,
+    'ui:option': { [string]: any },
+  };
+
+  declare type DataType =
+    | 'string'
+    | 'number'
+    | 'boolean'
+    | 'integer'
+    | 'array'
+    | '*'
+    | null;
+
+  declare type MatchType = {
+    types: Array<DataType>,
+    widget?: string,
+    field?: string,
+    format?: string,
+    $ref?: boolean,
+    enum?: boolean,
+  };
+
+  declare type FormInput = {
+    displayName: string,
+    // given a data and ui schema, determine if the object is of this input type
+    matchIf: Array<MatchType>,
+    // allowed keys for ui:options
+    possibleOptions?: Array<string>,
+    defaultDataSchema: {
+      [string]: any,
+    },
+    defaultUiSchema: {
+      [string]: any,
+    },
+    // the data schema type
+    type: DataType,
+    // inputs on the preview card
+    cardBody: React.AbstractComponent<{
```

Comment:
React is not imported here, so either import it or use le global type `React$AbstractComponent`